src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java

```diff
@@ -19,7 +19,7 @@
     public HashMap<String, Integer> PlayerScores = new HashMap<String, Integer>();
 
     public String getGameName() {
-        return new String("MONEY DOOR GAME");
+        return new String("money door game");
```

Comment:
Let's make this all upper-case again, not a fan of the informality associated with the lower-case.